edx/edx-enterprise
# [ENT-451] Fix formatting nits on Enterprise landing page

**Description:**
This pull request fixes two nits:

1. The course final price (in dollars) should have two decimal places if not an integer. If an integer, it should have no decimal places.
2. If one hour of effort per week is required, a singular noun should be used to describe that single hour.

We fix point 1 using a quick integer check and string formatting operation. We fix point 2 by using `ungettext` instead of `ugettext` for this particular string.

**JIRA:** [ENT-451](https://openedx.atlassian.net/browse/ENT-451)

**Screenshot:**
<img width="701" alt="screen shot 2017-07-06 at 1 46 55 pm" src="https://user-images.githubusercontent.com/7773758/27925332-6105be92-6252-11e7-98e1-928953302530.png">


**Dependencies:** None 

**Merge deadline:** 16 July 2017

**Installation instructions:**
Requires course discovery/ecommerce
Requires configured Enterprise entitlement for course with discount
Requires course to be configured in Studio with one hour of effort per week.

**Testing instructions:**

1. Browse to the Enterprise course enrollment page (`http://localhost:8000/enterprise/176596e9-b93b-4a15-97eb-65d26ea309ea/course/course-v1%3AedX%2BDemoX%2BDemo_Course/enroll/`).
2. Click the View Course Details link.
3. Verify that the final price is presented without a decimal place if the discount results in an integer price, or with two decimal places if the discount results in any other price.
4. Verify that the Effort is described as "1 hour per week, per course".

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**
